dotcom-rendering/src/lib/useSDC.ts

```diff
@@ -15,7 +15,9 @@ const useSDC = <T>(
 	key: string,
 	fetcher: (baseUrl: string, payload: T) => Promise<ModuleDataResponse>,
 ): ModuleDataResponse | undefined => {
-	const { data, error } = useSWRImmutable(key, fetcher);
+	const { data, error } = useSWRImmutable(key, fetcher, {
```

Comment:
I think it's dangerous to call this function `useSWRImmutable` when it's in fact simply the standard one: https://swr.vercel.app/docs/revalidation#disable-automatic-revalidations

Reply:
Good spot!
The import is wrong. I will switch to using the actual `useSWRImmutable`, and do some further testing to ensure behaviour is the same